powerbi-docs/developer/embedded/move-to-production.md

```diff
@@ -56,19 +56,17 @@ If you run out of embed tokens while testing, you need to purchase a Power BI Em
 
 Once you create a capacity, you can assign your workspace to that capacity.
 
-All the workspaces that contain Power BI resources related to the embedded content (including datasets, reports, and dashboards), must be assigned to capacities. For example, if an embedded report and the dataset bound to it reside in different workspaces, both workspaces must be assigned to capacities.
+Each workspace that contains a Power BI artifact related to the embedded content (including datasets, reports, and dashboards) must be assigned to capacities. For example, if an embedded report and the dataset bound to it reside in different workspaces, both workspaces must be assigned to capacities.
 
 ### Assign a workspace to a capacity using a service principal
 
-To assign a capacity to a workspace using a[service principal](embed-service-principal.md), use the [Power BI REST API](/rest/api/power-bi/capacities/groups_assigntocapacity). When you're using the Power BI REST APIs, make sure to use the [service principal object ID](embed-service-principal.md).
+To assign a workspace to a capacity using a [service principal](embed-service-principal.md), use the [Power BI REST API](/rest/api/power-bi/capacities/groups_assigntocapacity). When you're using the Power BI REST APIs, make sure to use the [service principal object ID](embed-service-principal.md).
 
 ### Assign a workspace to a capacity using a master user
 
-Follow the steps below to assign a capacity to a workspace using a **master user**.
+You can also assign a workspace to a capacity from the settings of that workspace using a **master user**. The master user must have admin permissions to that workspace, and also capacity assignment permissions to that capacity.
```

Comment:
Not sure this wording is better? I think we can use the original wording and then add this sentence:

The master user must have admin permissions for that workspace.

Reply:
The reason for this PR was that I lost time because I didnt know about:

>and also capacity assignment permissions to that capacity. 

The master user needs two permissions set in two different places, so I think we should document that to be helpful for silly people like me 😆. I grabbed this verbatim from [the referenced doc](https://docs.microsoft.com/en-us/power-bi/admin/service-admin-premium-manage#assign-from-workspace-settings), and it mentions doing it via the settings so I thought, great, keep it the same as the other doc.

I reworded the start of the paragraph for copywriting; You'll notice in the article that we say 4 times "assign workspace to capacity" and then othertimes we say "assign capacity to workspace". Just bugged me that it wasn't consistent 😄.

Does this make sense? If you're not keen on the copywriting ok cool, but at a min I think we should mention *both* permissions.